website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -259,6 +321,67 @@ queries](/nomad/api-docs#blocking-queries) and [required ACLs](/nomad/api-docs#a
   - `OIDCClientSecret` `(string: "")` - The OAuth client secret
     configured with your OIDC provider.
 
+  - `OIDCClientAssertion` `(OIDCClientAssertion)` - Optionally send a signed JWT
```

Comment:
Same comments as above. At first I thought the parameters under Create and Update were identical and was going to suggest putting them in a partial so we don't copy/paste content... but then I noticed a few Create parameters that aren't in Update so there goes the partial idea at this time. I might address this later, when I'm updating front matter descriptions for search.

I'd prefer the API docs be generated from the code, and I think Education is moving in that direction.

Reply:
Could still put the common parameters in a partial, since the partial would just be a markdown list. 

These parameters are in Create but not Update:
- JWTValidationPubKeys
- JWKSURL
- JWKSCACert
- ExpirationLeeway
- NotBeforeLeeway
- ClockSkewLeeway
- VerboseLogging

So common parms in partial and those 7 after the included partial in Create. 
Also has me wondering if those 7 parms are supposed to be in update but docs never updated?? Hmmmm

Reply:
It would not surprise me one bit if they should actually be identical, but someone missed updating them because it's easy to get wrong! I'll double-check on this.

Reply:
Alrighty, I checked, and those params work with Update, too, so I think they were just an oversight.

db3e0eb179e92dad5440d77a851c219ec030ea62 makes the whole block into a partial.